tests/unit/aegra/test_mcp_auth.py

```
@@ -244,3 +272,196 @@ async def test_resolver_caches_resolved_oauth_token(self):
+                "client_secret": "csecret",
+                "scopes": ["read", "write"],
+            },
+        }
+
+        with patch("deep_agent.aegra.mcp_auth.httpx.AsyncClient") as mock_client:
+            mock_ctx = AsyncMock(post=AsyncMock(return_value=mock_response))
+            mock_client.return_value.__aenter__ = AsyncMock(return_value=mock_ctx)
+            mock_client.return_value.__aexit__ = AsyncMock(return_value=False)
+            token = await resolver.resolve("user-1", "cc-mcp", cfg)
+
+        assert token == "scoped-token"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "token"

(S105)